cloudhub-2/modules/ROOT/pages/ps-create-configure-tgw.adoc

```diff
@@ -10,6 +10,8 @@ You don't need to create the resource share again.
 * Ensure the private space belongs to the same region in which you are creating the transit gateway attachment. Inter-region peering is not supported.
 For example, you cannot attach transit gateway attachment from region A to a private space belonging to region B.
 
+IMPORTANT: Update your infrastructure (such as firewalls, routes, and connections) before creating transit gateway attachments. When a transit gateway is pending acceptance, all infrastructure updates are delayed until you accept the attachment.
```

Comment:
IMPORTANT: When a transit gateway is pending acceptance, all infrastructure updates (such as updates to firewalls, routes, and connections) are delayed until you accept the attachment. Changes to infrastructure do not take effect until the pending transit gateway request is accepted, so you must accept the pending acceptance request as soon as possible or delay making infrastructure changes until there are no pending requests.

Reply:
```suggestion
IMPORTANT: When a transit gateway is pending acceptance, all infrastructure updates (such as updates to firewalls, routes, and connections) are delayed until you accept the attachment. Changes to infrastructure do not take effect until the pending transit gateway request is accepted, so you must accept the pending acceptance request as soon as possible or delay making infrastructure changes until there are no pending requests.
```